buildscripts/incremental/test.cmd

```diff
@@ -1,4 +1,4 @@
 
 call activate %CONDA_ENV%
 
-python runtests.py -v
+python runtests.py
```

Comment:
This change should be undone (the `-v` flag prints out the test name before it runs, which is very useful for seeing what the interpreter was approximately doing if it dies / hangs in a test).

Reply:
yep, sorry, I forgot I had changed this to see if I could get more info. I will change it back.